Tests/SwiftDocCTests/Test Bundles/TestBundle.docc/mykit-iOS.symbols.json

```diff
@@ -552,6 +552,11 @@
       "source" : "s:5MyKit0A5ClassCACycfc",
       "target" : "s:5MyKit0A5ClassC",
       "kind" : "memberOf"
+    },
+    {
```

Comment:
This duplicated symbol didn't have any relationships so it wasn't a member of anything. It makes the most sense to have a copy of the original's relationship.